leaflet-vue-amplify/vite.config.js

```diff
@@ -11,6 +11,9 @@ export default defineConfig({
       "@": fileURLToPath(new URL("./src", import.meta.url)),
```

Comment:
Please add this line otherwise the `npm run build` fails, after the edit the `resolve.alias` should look like this:
```js
alias: {
      "@": fileURLToPath(new URL("./src", import.meta.url)),
      "./runtimeConfig": "./runtimeConfig.browser",
    },
```

Reply:
And also if you could do the same with the other two samples that have `vue` in the name